integreat_cms/cms/models/external_calendars/external_calendar.py

```diff
@@ -0,0 +1,71 @@
+import icalendar
+import requests
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from integreat_cms.cms.models.abstract_base_model import AbstractBaseModel
+from integreat_cms.cms.models.regions.region import Region
+
+
+class ExternalCalendar(AbstractBaseModel):
+    """
+    Model for representing external calendars, from which events can be imported.
+    """
+
+    name = models.CharField(max_length=255, verbose_name=_("calendar name"), default="")
+    region = models.ForeignKey(
+        Region,
+        on_delete=models.CASCADE,
+        verbose_name=_("region"),
+        related_name="external_calendars",
+    )
+    url = models.URLField(max_length=250, verbose_name=_("URL"))
+
+    import_filter_tag = models.CharField(
+        max_length=255,
+        blank=True,
+        default=settings.EXTERNAL_CALENDAR_TAG,
+        verbose_name=_(
+            "The tag that events need to have to get imported (Leave blank to import all events)"
+        ),
```

Comment:
I wonder whether we should rename this also for the user or if *tags* is originating somewhere which would make this more familiar and intuitive to the user…?

```suggestion
        verbose_name=_(
            "The category that events need to have to get imported (Leave blank to import all events)"
        ),
    )
```